backend/server/controllers/posts.js

```diff
@@ -166,7 +164,7 @@ export const deletePostById = async (req, res) => {
                     if (count !== 0) {
                         res.status(200).send('The message has been deleted.')
                     } else {
-                        res.status(400).send('Failed to update.')
+                        res.status(400).send('Failed to delete.')
```

Comment:
I think this should be 500, because if we failed to delete it for an unknown reason then it's a server error rather than a client error

Also I just realised, the if statement above it should say "The post has been deleted" right?

Reply:
Actually, I think 400 is correct. It's the error message that is incorrect. The error occurs when the id for that particular post can not be found.

Reply:
Wait nvm, that should be 404 not found.

Reply:
But you already checked if the post exists when you typed `if (!post) {`

Reply:
yeah i just realised

Reply:
I made the change now